mass_editing/wizard/mass_editing_wizard.py

```diff
@@ -22,6 +22,11 @@ def _prepare_fields(self, line, field, field_info):
                 ("remove_m2m", _("Remove")),
                 ("add", _("Add")),
             ]
+        elif field.ttype == "one2many":
+            selection = [
```

Comment:
```suggestion
            selection = [("set_o2m", _("Set")), ("remove_o2m", _("Remove"))]
```

Maybe this way coverage doesn't decrease.

Reply:
Thanks @cubells , I have applied the change that you have suggested but Travis gives an error, I don't know why. I have left it as it was before although it still shows:

    mass_editing/wizard/mass_editing_wizard.py#L26
    Added line #L26 was not covered by tests